### PR TITLE
v1 API

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rokamura@yahoo-corp.jp>

merge this PR after https://github.com/vdaas/vald/pull/740 is merged

### CODE REVIEW

File: example-ts/example.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import grpc = require('grpc');
+import { v1_vald, v1_payload } from 'vald-client-node';
+
+// create clients
+
+let iclient = new v1_vald.insert_grpc.InsertClient('localhost:8081', grpc.credentials.createInsecure());
+let sclient = new v1_vald.search_grpc.SearchClient('localhost:8081', grpc.credentials.createInsecure());
```
[the remark this answers is not in the document]
Comment (rinx) [reply example-ts/example.ts:7]: i see. thanks :+1:

File: example-ts/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "example-ts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "example.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
[the remark this answers is not in the document]
Comment (rinx) [reply example-ts/package.json:10]: maybe MIT is better. thanks!

File: example/example.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+sreq.setConfig(scfg);
+
+sclient.search(sreq, function (err, resp) {
+    if (err != null) {
+        console.log('error: ', err);
+    } else {
+        // resp is Search.Response
+        console.log('resp: ', resp);
+    }
+});
```
[the remark this answers is not in the document]
Comment (rinx) [reply example/example.js:58]: i'll revise it later. thanks :+1:

File: example/package.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "example.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (rinx) [inline example/package.json:10]: revise license.

File: src/google/api/annotations_pb.d.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+// package: google.api
+// file: google/api/annotations.proto
+
+import * as jspb from "google-protobuf";
+import * as google_api_http_pb from "../../google/api/http_pb";
+import * as google_protobuf_descriptor_pb from "google-protobuf/google/protobuf/descriptor_pb";
+
+  export const http: jspb.ExtensionFieldInfo<google_api_http_pb.HttpRule>;
```
[the remark this answers is not in the document]
Comment (rinx) [reply src/google/api/annotations_pb.d.ts:8]: this is a generated code by protoc. so we'll don't have to care about it i think.

File: example/example.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (rinx) [reply example/example.js]: revised. thanks for your comment.

**Review by vankichi — APPROVED** [review]
Thanks.
LGTM 🙆‍♂️

**Pull request merged**